### ISSUE TITLE
"See all" toggled when the remaining feed of a category is set to seen

### ISSUE THREAD

**Issue by ZeHiro** [FreshRSS/FreshRSS#971]
To reproduce : 
- Feedly is set up to only show unread articles
- Several feeds in a given category
- only one feed has unread article
- click "set read" for the feed

Observed : 
- The feed is set as read
- The menu shows all the feeds of the category
- The main view (listing the articles) displays read articles
- The active categoryis unchanged
- The option "See all, including read" option is set globally
  Expected : 
- The displayed categoryin the menu is the next categoryin the list
- The "see all, including read" shouldn't be active. 

If the action "set read" is done on the category, and not the feed, the behaviour is the expected one

**Comment by Alkarex** [comment]
Yes, this is due to the mechanism that goes through all unread feeds of a given category, and adds this step of showing the full category when done, so that the user knows when all the feeds of a given category are read. So it is on purpose, but I agree that it could maybe be done better.

**Comment by marienfressinaud** [comment]
I already observed this behaviour and wanted to change it: I expect that we go to the next unread feed (not category)

**Comment by math-GH** [comment]
Is it still relevant?
